run-orachk.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+orachk_dir=$(dirname $0)
+orachk_env_file=${orachk_dir}/orachk.sh
+
+[[ -f $orachk_env_file ]] || {
+    echo
+    echo $orachk_env_file not found
+    echo
+    exit 1
+}
+
+. $orachk_env_file
+
+# use of undeclared variables is fatal
+set -u
```

Comment:
Maybe  'set -e' too, to make errors in subcommands fatal too?

Reply:
The scrips that had 'set -u' no longer exist.